src/modules/client/components/clientAlerts/ClientAlertCard.tsx

```diff
@@ -30,32 +30,34 @@ const ClientAlertCard: React.FC<ClientAlertCardProps> = ({
 
   return (
     <TitleCard title={title} headerVariant='border'>
-      <Box sx={{ m: 2 }}>
-        {loading && <Loading />}
-        {!loading && clientAlerts.length === 0 && (
-          <Stack
-            direction='row'
-            justifyContent='space-between'
-            alignItems='center'
-            p={1}
-            sx={{
-              backgroundColor: (theme) => theme.palette.grey[50],
-              color: 'text.secondary',
-              borderRadius: 1,
-            }}
-          >
-            <Typography variant='body2'>
-              {alertContext} has no alerts at this time
-            </Typography>
-            {children}
-          </Stack>
-        )}
-        {clientAlerts.length > 0 && (
-          <ClientAlertStack clientAlerts={clientAlerts}>
-            {children}
-          </ClientAlertStack>
-        )}
-      </Box>
+      {loading && <Loading />}
```

Comment:
This is an unrelated style change for consistency with the other components loading on this page. The contents of the component are not changing, it's just moving the Loading indicator outside the Box, and wrapping the whole Box in a conditional so it only shows when loading is false.

before:
![Screenshot 2024-07-29 at 8 36 32 AM (2)](https://github.com/user-attachments/assets/c6c744df-a7f8-49b1-b402-98a16842156f)

after:
![Screenshot 2024-07-29 at 8 40 36 AM (2)](https://github.com/user-attachments/assets/0a23a887-3932-4f86-ad10-54bc36d62139)